pkgs/by-name/ct/ctune/package.nix

```diff
@@ -0,0 +1,64 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  openssl,
+  curl,
+  ffmpeg,
+  vlc,
+  SDL2,
+  lame,
+  json_c,
+  cmake,
+  pkg-config,
+  ncurses,
+  libuuid,
+  pandoc,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "ctune";
+  version = "1.3.2";
+
+  src = fetchFromGitHub {
+    owner = "An7ar35";
+    repo = "ctune";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-fy7Wb4R6HDa+cLxI5YWq67gxw1XS/iDDN4M3Rn7GYeI=";
+  };
+
+  nativeBuildInputs = [
```

Comment:
Since you already went to the effort of separating deps here, might as well set `strictDeps = true;`